[NETBEANS-54] Module Review j2ee.core.utilities

juneau001 opened the pull request:
Updated j2ee.core.utilities license headers, where needed.
- no external libraries
- checked Rat report and adjusted license headers accordingly on identified problem files
- skimmed module and did not see any other issues

matthiasblaesing commented:
Please have another look at this:
- Please ignore the files covered in th "Problems to be solved centrally" of https://cwiki.apache.org/confluence/display/NETBEANS/List+of+Modules+to+Review. For example the .form files were updated centrally from their corresponding .java files. These changes need to removed from this changeset. I suggest you also remove changes that relate to file in the mentioned section.
- the latter change might also fix the changes line endings in build.xml

juneau001 commented:
Thanks for the feedback and sorry for missing those "Problems to be solved centrally".

junichi11 commented:
#21 was merged. So, it seems that this PR includes the commit for removing the license header for .form files.

matthiasblaesing commented:
@juneau001 please rereview this changeset - now your PR would remove headers from .form files. While your at it: rebase your changes onto the current HEAD of master and squash the changes together.

juneau001 commented:
@matthiasblaesing Thanks for the note.  Very sorry for causing an issue, I thought that you wanted me to remove the license headers from the .form files, per the "Problems to be solved centrally" of https://cwiki.apache.org/confluence/display/NETBEANS/List+of+Modules+to+Review as I originally had added license headers to these files.  I believe that I also should remove the license headers from the project.properties, build.xml, and build-impl.xml since they should also be solved centrally...so I should back out all changes that I made since each of the files was in that list?  I appreciate the assistance.

matthiasblaesing commented:
The problem is that you added the license headers, that conflicted with the central change the @jlahoda kindly did (fixing a great many files). Now your changeset removes these headers again. On github you can see in the "Files changed" section what would happen if your PR is merged and in this case the correct headers are removed for form files.

So:
- please rebase your changes onto master
- please revert your changes to the form files
- please also revert the change to genfiles.properties (See: https://github.com/apache/incubator-netbeans/pull/46/commits/40a715ad91e47696a8018c48c7ad091b9c4341ec) as that is a generated file

The rest looks good. The whitespace changes in build.xml are the removal of trailing spaces, which I think is ok.

matthiasblaesing commented:
Oh - and no reason for apologies! The work is appreciated.

juneau001 commented:
Hi @matthiasblaesing is the code looking okay now?  I've done what was mentioned.  That is, I rebased changes on to master, reverted my changes to form files and genfiles.properties.  Let me know how things look now.  Thanks

emilianbold commented on j2ee.core.utilities/src/org/netbeans/modules/j2ee/core/support/java/method/ExceptionsPanel.form:
```diff
@@ -1,26 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
-<!--
```
This should be reverted.

juneau001 replied:
Thanks @emilianbold, can I ask why to revert the .form changes?  I took the license headers out because these are auto-generated and I mistakingly put them in the first time.

emilianbold replied:
In the mean time the decision was to put them in. And they were added all over the codebase automatically, via a script.

So now your commit appears to remove what was just added.

juneau001 replied:
I'm sorry about that.  I am thinking that I will close this pull request and start fresh...sound ok?  Thanks

emilianbold replied:
Don't. Just start in another branch on your local machine then push that into your master branch on github (eventually with --force). The existing PR will pick the updated changesets and we can go from there.

emilianbold commented on j2ee.core.utilities/src/org/netbeans/modules/j2ee/core/support/java/method/MethodCustomizerPanel.form:
```diff
@@ -1,26 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
-<!--
```
This should also be reverted.

emilianbold commented on j2ee.core.utilities/src/org/netbeans/modules/j2ee/core/support/java/method/ParametersPanel.form:
```diff
@@ -1,26 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
-<!--
-
```
... and this.

emilianbold commented on j2ee.core.utilities/test/unit/data/JavaApp/build.xml:
```diff
@@ -1,4 +1,25 @@
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
```
This is OK.

emilianbold commented on j2ee.core.utilities/test/unit/data/JavaApp/build.xml:
```diff
@@ -7,9 +28,9 @@
-    used for execution of your tasks. These targets are usually executed 
-    before and after some main targets. They are: 
+    There exist several targets which are by default empty and which can be
+    used for execution of your tasks. These targets are usually executed
```
These whitespace changes are not necessary. I generally prefer the patch to contain only the fix, not reformatting.

emilianbold commented on j2ee.core.utilities/test/unit/data/JavaApp/build.xml:
```diff
@@ -35,20 +56,20 @@
         </target>
 
-    For list of available properties check the imported 
-    nbproject/build-impl.xml file. 
```
More whitespace.

emilianbold commented on j2ee.core.utilities/test/unit/data/JavaApp/build.xml:
```diff
@@ -35,20 +56,20 @@
 
     Another way to customize the build is by overriding existing main targets.
-    The targets of interest are: 
+    The targets of interest are:
```
More whitespace.

emilianbold commented on j2ee.core.utilities/test/unit/data/JavaApp/build.xml:
```diff
@@ -35,20 +56,20 @@
       -do-jar-with-manifest:    JAR building (if you are using a manifest)
       -do-jar-without-manifest: JAR building (if you are not using a manifest)
-      run:                      execution of project 
+      run:                      execution of project
```
More whitespace.

emilianbold commented on j2ee.core.utilities/test/unit/data/JavaApp/build.xml:
```diff
@@ -60,10 +81,10 @@
 
-    Notice that the overridden target depends on the jar target and not only on 
-    the compile target as the regular run target does. Again, for a list of available 
+    Notice that the overridden target depends on the jar target and not only on
```
More whitespace.

emilianbold commented on j2ee.core.utilities/test/unit/data/JavaApp/nbproject/build-impl.xml:
```diff
@@ -1,5 +1,25 @@
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
```
This is OK.

emilianbold commented on j2ee.core.utilities/test/unit/data/JavaApp/nbproject/build-impl.xml:
```diff
@@ -21,9 +41,9 @@ is divided into following sections:
         -->
 <project xmlns:j2seproject1="http://www.netbeans.org/ns/j2se-project/1" xmlns:j2seproject2="http://www.netbeans.org/ns/j2se-project/2" xmlns:j2seproject3="http://www.netbeans.org/ns/j2se-project/3" xmlns:jaxrpc="http://www.netbeans.org/ns/j2se-project/jax-rpc" basedir=".." default="default" name="JavaApp-impl">
     <target depends="test,jar,javadoc" description="Build and test whole project." name="default"/>
-    <!-- 
```
More whitespace.

emilianbold commented on j2ee.core.utilities/test/unit/data/JavaApp/nbproject/project.properties:
```diff
@@ -1,3 +1,20 @@
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
```
OK

emilianbold commented on j2ee.core.utilities/test/unit/data/JavaApp/nbproject/project.xml:
```diff
@@ -1,4 +1,24 @@
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
```
OK.
